packages/oas-kit-common/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oas-kit-common",
-  "version": "1.0.6",
+  "version": "1.0.7",
```

Comment:
Correcting these package version numbers should be an automatic process. Is it handled by a `lerna` version higher than the one we install in `package.json`?